app/javascript/services/checkDomainsInFile.jsx

```diff
@@ -1,20 +1,15 @@
 import apiRequest from "./api/apiRequest";
 
 const checkDomainsInFile = async (file) => {
-  let data = new FormData();
-  data.append("file", file);
   /// Send the file to the api to analyze
   const response = await apiRequest({
     url: "/api/v1/specifications/info",
     method: "post",
-    payload: data,
+    payload: {
+      file: JSON.stringify(file, null, 2),
```

Comment:
Out of curiosity: why does it need to have white space? And can't you send the file's content as-is, without stringifying it?

Reply:
Corrected! Actually, it doesn't need spaces nor a replacer.

But it needs to be stringified because the backend processes the parameter with `JSON.parse()` and removing both the stringify and the parse methods (back and front) won't work even trying with headers: `{ "Content-Type": "application/json" }`.